<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>코딩스튜디오 - 오픈월드 프로젝트</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnShowTileMap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>타일맵보이기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowTileLine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>타일선보이기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowTileMapLine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>타일맵/선보이기</string>
    </property>
   </widget>
   <widget class="OWDrawWidget" name="widgetOpenWorld" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowDataMap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>타일데이타보이기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>배경색</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboBackgroundColor">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Open World 2D Engine Editor</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>마우스 이벤트</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매트릭의 데이타</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매트릭의 픽셀 위치</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매트릭 위치</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtMetricLocation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtMetricPixel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtMetricData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OWDrawWidget</class>
   <extends>QWidget</extends>
   <header>owdrawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
